cmake/assets/run_nodered_template.sh.in

```diff
@@ -1,3 +1,17 @@
-cd @EVEREST_DOCKER_DIR@
-docker build -t everest-nodered .
-docker run --rm --network host --name everest_nodered --mount type=bind,source=@FLOW_FILE@,target=/data/flows.json everest-nodered
+# In the following a volume is created to contain the nodered config, this is done to
+# allow starting the nodered container from inside a devcontainer
+
+# Create docker volume to contain nodered config
+docker volume create nodered-config-volume
```

Comment:
`nodered-config-volume` is quite generic, not sure if it might collide with other node red instances.  Would it be possible to use a local path inside `build` for example which gets mounted into the container instead of an volume.  This would also ease the access to the config file.